### ISSUE TITLE
0 background in the lowest energies

### ISSUE THREAD

**Issue by kazumaishio** [gammapy/gammapy#3981]
**Gammapy version**
gammapy: 0.20
numpy: 1.21.5
astropy: 5.0.4
python: 3.8.12

**Bug description**
0 background in the lowest energies

I follow the tutorial:
https://docs.gammapy.org/0.20/tutorials/analysis/1D/cta_sensitivity.html
and draw sensitivity curve the irf:
Prod5-North-60deg-NorthAz-4LSTs.1800s-v0.2.fits.gz
which is available here:
https://gitlab.cta-observatory.org/cta-science/simulations/public-products/public-irfs/-/tree/prod5-v0.2-analysis

The Table returned from the sensitivity_estimator.run() shows weird values in the lowest energies. 
The first three energy bins are with no background. 
(I am no the way to see if this happens in other files)

**Expected behavior**
The background should be dominant in the lowest energies.

**To Reproduce**

inputpath="IRFs-fits/prod5-v0.2-IRFs/fits/Prod5-North-60deg-NorthAz-4LSTs.1800s-v0.2.fits.gz"

import matplotlib.pyplot as plt
import numpy as np
import astropy.units as u
from astropy.coordinates import Angle, SkyCoord

from gammapy.irf import load_cta_irfs
from gammapy.makers import SpectrumDatasetMaker
from gammapy.data import Observation
from gammapy.estimators import SensitivityEstimator
from gammapy.datasets import SpectrumDataset, SpectrumDatasetOnOff
from gammapy.maps import MapAxis, RegionGeom

energy_axis = MapAxis.from_energy_bounds("0.03 TeV", "30 TeV", nbin=20)
energy_axis_true = MapAxis.from_energy_bounds(
    "0.01 TeV", "100 TeV", nbin=100, name="energy_true"
)

geom = RegionGeom.create("icrs;circle(0, 0.5, 0.1)", axes=[energy_axis])

empty_dataset = SpectrumDataset.create(
    geom=geom, energy_axis_true=energy_axis_true
)
irfs = load_cta_irfs(inputpath)
pointing = SkyCoord("0 deg", "0 deg")
obs = Observation.create(pointing=pointing, irfs=irfs, livetime="5 h")
spectrum_maker = SpectrumDatasetMaker(
    selection=["exposure", "edisp", "background"]
)
dataset = spectrum_maker.run(empty_dataset, obs)

containment = 0.68
# correct exposure
dataset.exposure *= containment
# correct background estimation
on_radii = obs.psf.containment_radius(
    energy_true=energy_axis.center, offset=0.5 * u.deg, fraction=containment
)
factor = (1 - np.cos(on_radii)) / (1 - np.cos(geom.region.radius))
dataset.background *= factor.value.reshape((-1, 1, 1))
dataset_on_off = SpectrumDatasetOnOff.from_spectrum_dataset(
    dataset=dataset, acceptance=1, acceptance_off=5
)
sensitivity_estimator = SensitivityEstimator(
    gamma_min=10, n_sigma=5, bkg_syst_fraction=0.10
)
sensitivity_table = sensitivity_estimator.run(dataset_on_off)
print(sensitivity_table)

**Comment by AtreyeeS** [comment]
Yes indeed! As @adonath guessed, the IRF is empty in these bins for the specific case @kazumaishio mentions. Since extrapolations of IRFs are not performed by gammapy anymore, the behaviour that you see is expected. Probably an older version of gammapy were used to make the  CTA sensitivity curves
